sharepoint/sharepoint-ps/sharepoint-online/Add-SPOContentSecurityPolicy.md

```diff
@@ -0,0 +1,75 @@
+---
+external help file:
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/add-spocontentsecuritypolicy
+applicable: SharePoint Online
+title: Add-SPOContentSecurityPolicy
+schema: 2.0.0
+author: jaobrie
+ms.author: jaobrie
+manager: ryannak
+ms.reviewer:
+---
+
+# Add-SPOContentSecurityPolicy
+
+## SYNOPSIS
+
+Adds an entry to the Content Security Policy configuration.
+
+## SYNTAX
+
+### Default
+
+```powershell
+Add-SPOContentSecurityPolicy [-Url] <String> [-Directive] <String> 
+```
+
+## DESCRIPTION
+
+Adds an entry to the Content Security Policy configuration. 
+The url in each entry will be added to the corresponding directive during construction of the Content-Security-Policy header.
+Entries with a "*" directive will be applied to all directives.
+
```

Comment:
I would add: "in multi geo environments, Content Security Policy entries are different for each geo" (or something on that line)